### PR TITLE
feat: re-enabling billing capture in support bundle

### PR DESCRIPTION
**Pull request by Elsie4ever**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

File: azext_edge/tests/edge/support/create_bundle_int/helpers.py [hunk 1]
```diff
@@ -262,9 +262,9 @@ def _get_namespace_determinating_files(
         level_1 = walk_result.pop(path.join(BASE_ZIP_PATH, clusterconfig_namespace))
         assert level_1["folders"] == ["clusterconfig"]
         assert not level_1["files"]
-        level_2 = walk_result.pop(path.join(BASE_ZIP_PATH, namespace, "clusterconfig"))
-        assert level_2["folders"] == ["billing"]
-        assert not level_2["files"]
+        # level_2 = walk_result.pop(path.join(BASE_ZIP_PATH, namespace, "clusterconfig"))
```
Comment (digimaun) [inline azext_edge/tests/edge/support/create_bundle_int/helpers.py:265]: What was the reason this is commented out ?

Comment (Elsie4ever) [reply azext_edge/tests/edge/support/create_bundle_int/helpers.py:265]: I was a change I forgot to remove, will update!

Comment (Elsie4ever) [reply azext_edge/tests/edge/support/create_bundle_int/helpers.py:265]: no more clusterconfig under aio namespace

File: azext_edge/tests/edge/support/test_support_unit.py [hunk 2]
```diff
@@ -645,6 +662,10 @@ def assert_list_pods(
                     "namespace: namespace\ntimestamp: '0000-00-00T00:00:00Z'\n",
                 )
 
+            if "prefix_names" in kwargs:
+                if pod_name not in kwargs["prefix_names"]:
```
Comment (vilit1) [inline azext_edge/tests/edge/support/test_support_unit.py:666]: could be one if rather than nested

Comment (vilit1) [reply azext_edge/tests/edge/support/test_support_unit.py:666]: tbh you can even get away with simplifying further with a kwargs.get()

**Review by digimaun — APPROVED** [review]

**Comment by Elsie4ever** [comment]
latest IT runs
image

**Pull request merged**